lib/vagrant-openshift/action.rb

```diff
@@ -174,6 +188,18 @@ def self.run_origin_tests(options)
           b.use TestExitCode
         end
       end
+      
+      def self.run_origin_asset_tests(options)
+        Vagrant::Action::Builder.new.tap do |b|
+          # UI integration tests require the api server to be running
+          b.use RunSystemctl, {:action => "restart", :service => "openshift"}        
```

Comment:
start should be fine, the other thing i debated is whether i should do a stop after the tests have run